client/packages/invoices/src/utils.ts

```diff
@@ -23,6 +23,16 @@ import {
   SupplierReturnRowFragment,
 } from './Returns';
 
+export const allStatuses: InvoiceNodeStatus[] = [
```

Comment:
I made this to separate StatusCrumbs from createStatusLog (the latter needs all statuses, otherwise statusIdx = 0, and the former just needs subset of statuses for display)